index.html

```diff
@@ -7,5 +7,7 @@ <h1>Hello Microverse!</h1>
     <h3> Day 1 of Microverse </h3>
```

Comment:
- [ ] I would kindly suggest you make this part an `h1` tag as this is the main header of the webpage. 👍 👍